### PR TITLE
improve sample certificate creation Makefile

### PR DESCRIPTION
**Pull request by elevran**
- create a root-ca config
- leave intermediate files to save work when creating multiple intermediate certificates

### CODE REVIEW

File: samples/certs/Makefile [hunk 1]
```diff
@@ -1,18 +1,26 @@
 .SUFFIXES: .csr .pem .conf
 .PRECIOUS: %/ca-key.pem %/ca-cert.pem %/cert-chain.pem
-#.PRECIOUS: %/cluster-ca.csr %/intermediate.conf
+.PRECIOUS: root-cert.csr root-ca.conf %/cluster-ca.csr %/intermediate.conf
```
Comment (ymesika) [inline samples/certs/Makefile]: You have '.PRECIOUS` above. Isn't it overriding it?

Comment (elevran) [reply samples/certs/Makefile]: No, they are additive. Same as you can have multiple `.PHONY` lines

**Pull request closed without merging**